python/xrprimer/transform/convention/world/smc_world.py

```diff
@@ -0,0 +1,12 @@
+from .base_world import BaseWorld
+
+
+class SenseMoCapWorld(BaseWorld):
+    """A world class for SenseMoCap coordinate convention.
```

Comment:
What about renaming it to `XRMoCap`? So as the filename from `smc` to `xmc`

Reply:
Like XRPrimer, XRMoCap is also a code base that is compatible with multiple definitions of world coordinate systems: panoptic, smc, campus, etc. `SenseMoCapWorld` is only for `.smc` files and other data like it.